packages/expo/src/effects/NativeLinearGradient.web.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { View, StyleSheet } from 'react-native';
+
+type Props = {
+  colors: number[];
+  locations?: number[] | null;
+  startPoint?: Point | null;
+  endPoint?: Point | null;
+  onLayout?: Function;
+} & React.ElementProps<View>;
+
+type State = {
+  width: number;
+  height: number;
+};
+type Point = [number, number];
+
+export default class NativeLinearGradient extends React.PureComponent<Props, State> {
+  state = {
+    width: 0.5,
+    height: 0.5,
+  };
+
+  onLayout = event => {
+    this.setState({
+      width: event.nativeEvent.layout.width,
+      height: event.nativeEvent.layout.height,
+    });
+    if (this.props.onLayout) {
+      this.props.onLayout(event);
+    }
+  };
+
+  get angle(): string {
```

Comment:
```suggestion
  getAngle(): string {
```

Use a method instead of a getter so that it's clear to the caller that there's some actual work done here.